src/pages/attractions-facilities.html

```diff
@@ -1,222 +1,400 @@
+              <img src="../public/images/facilities-2.jpg" style="width:100%">
+            </div>
+            
+            <div class="article-3-mySlides article-fade">
+              <img src="../public/images/facilities-3.jpg" style="width:100%">
+            </div>
+            
+            <a class="article-prev-3" onclick="article_plus_slides_3(-1)">❮</a>
+            <a class="article-next-3" onclick="article_plus_slides_3(1)">❯</a>
+
+          </div>
+            
+          <div class="flex-child">
 
-            <a href="facebook.com">Facebook</a> |
-            <a href="twitter.com">Twitter</a> |
-            <a href="instagram.com">Instagram</a>
+            <div class="flip-card" style="margin-top: 6.25%; margin-left: 12.5%;">
 
-            <div>
-            <span>Contacts</span>
-            <span>One</span>
-            <span>Two</span>
+              <div class="flip-card-inner">
+            
+                <div class="flip-card-front" style="background-color: #2980b9;">
+                  <p>Facilities</p>
+                </div>
+            
+                <div class="flip-card-back" style="background-color: #2980b9;">
+                  <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                    sed do eiusmod tempor incididunt ut labore et dolore 
+                    magna aliqua. Ut enim ad minim veniam, quis nostrud 
+                    exercitation ullamco laboris nisi ut aliquip ex ea 
+                    commodo consequat. Duis aute irure dolor in reprehenderit 
+                    in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+                  </p> 
+                </div>
+            
+              </div>
+            
             </div>
 
+          </div>
+            
+        </div>
+
+        <!--Contact details, park map, ticket info-->
+        <div class="nth_shape_container_div">
+
+          <div class="flex-child">
+            <img src="../public/images/circle.png" style="width:75%">
+          </div>
+
+          <div class="flex-child">
+            <img src="../public/images/triangle.png" style="width:75%">
+          </div>
+
+          <div class="flex-child">
+            <img src="../public/images/square-2.png" style="width:75%">
+          </div>
+            
+        </div>
+
+        <!-- footer -->
+        <footer>
+              <div class="flex-child">
+                  <a href="#" class="fa fa-facebook"></a>
+                  <a href="#" class="fa fa-twitter"></a>
+                  <a href="#" class="fa fa-instagram"></a>
+              </div>
+              <div class="flex-child">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit
+              </div>
+              <div class="flex-child">
+                  Consectetur adipiscing elit
+              </div>
         </footer>
+      </div>
+
+      <!--header carousel javascript-->
+      <script>
+      let slideIndex = 0;
```

Comment:
Move this script to `public/scripts` and reference it on `<script src="../public/scripts/attractions-facilities-slider.js"></script>